explorer/pages/[entityListType]/index.tsx

```diff
@@ -42,9 +42,7 @@ const seedDatabase = async function seedDatabase( // TODO get rid of this duplic
   }
 
   const object = JSON.parse(rawData.toString());
-  const entities = entityMapper
-    ? Object.values(object).map(entityMapper)
-    : Object.values(object);
+  const entities = Object.values(object); // Client-side fetched entities are mapped prior to dispatch to explore state.
```

Comment:
Client-side fetched entities are mapped prior to dispatching the entities to explore state. Removing the mapping from the database to prevent mapping over entities twice!